### ISSUE TITLE
where can I download gmdepth-scale1-resumeflowthings-scannet-5d9d7964.pth?

### ISSUE THREAD

**Issue by wanmeihuali** [donydchen/mvsplat#2]

**Comment by donydchen** [comment]
Hi @wanmeihuali, I hope you have found it as detailed in the README.md. I reply and copy it here for the potential future reference of others.

```bash
# download the backbone pretrained weight from unimath and save to 'checkpoints/'
wget 'https://s3.eu-central-1.amazonaws.com/avg-projects/unimatch/pretrained/gmdepth-scale1-resumeflowthings-scannet-5d9d7964.pth' -P checkpoints
```

**Comment by wanmeihuali** [comment]
Thanks! Maybe you can add it to the evaluation section.

**Comment by donydchen** [comment]
Temporary fixed with 4d3901e37a078a78022775cf46b9c9ece21e3c04.
